backend/src/firebase-configs/forgettable_service_account.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Might want to remove this file but otherwise I think it all looks good!

Reply:
Agreed, the service account shouldn't be publicly available.

Reply:
True, I'll remove this and the front-end files